service/api_patch.go

```diff
@@ -58,6 +58,11 @@ func (as *APIServer) submitPatch(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	if data.Alias == evergreen.CommitQueueAlias && !patch.IsMailboxDiff(patchString) {
```

Comment:
I think disallowing non-mbox patches means [the check](https://github.com/evergreen-ci/evergreen/blob/d05b5230f7daddf685681e4634b6b3844c5531ea/service/api_patch.go#L165) added in 2e06d58f2b0ac612cf4771c019e0f3176cd03d0d could be replaced by disallowing non-mbox commit queue patches in set-module

Reply:
is this worthwhile? 
Although the check in set-module should be unnecessary for making sure the user isn't using an old CLI (if they managed to create a commit queue patch in the first place they must be using a post-mbox CLI), it could be a cleaner solution to the problem addressed in 2e06d58.

Reply:
Sorry, I missed this comment! Good catch